src/borg/logger.py

```diff
@@ -165,24 +165,38 @@ def setLevel(self, *args, **kw):
             return self.__logger.setLevel(*args, **kw)
 
         def log(self, *args, **kw):
+            if 'msgid' in kw:
+                kw.setdefault('extra', {})['msgid'] = kw.pop('msgid')
```

Comment:
Adding a function and pasting one line to all of these versus pasting two lines to all of these -- not much difference imho.